make/common/Modules.gmk

```diff
@@ -231,6 +231,16 @@ ifeq ($(call isTargetOs, windows macosx linux), false)
   MODULES_FILTER += jdk.incubator.jpackage
 endif
 
+################################################################################
+# Filter out specific modules for mobile platforms
+
+ifeq ($(call isTargetOs, ios android), true)
+  MODULES_FILTER += jdk.hotspot.agent
```

Comment:
You don't need to do this since you turn off SA. There is already a 
`
ifeq ($(INCLUDE_SA), false)
  MODULES_FILTER += jdk.hotspot.agent
endif
`